src/app/components/toggle-switch/index.html

```diff
@@ -0,0 +1,77 @@
+<sky-demo-page
+  pageTitle="Toggle switch"
+  navTitle="Toggle switch"
+  skyuxModule="SkyToggleSwitchModule"
+  packageName="@skyux/forms"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The toggle switch component is useful when a value can either be "on" or "off". When users interact with a toggle switch, its value is driven through an <sky-code>ngModel</sky-code> attribute that you specify on the <sky-code>sky-toggle-switch</sky-code> element.
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties
+    sectionHeading="Toggle switch properties"
+  >
+    <sky-demo-page-property
+      propertyName="ariaLabel"
+      defaultValue="undefined"
+      [isOptional]="true"
+    >
+      Defines a string value to label the toggle switch and set the <sky-code>aria-label</sky-code> attribute to support <a stacheRouterLink="." fragment="accessibility">accessibility</a>. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="checked"
+      defaultValue="false"
+      [isOptional]="true"
+    >
+      Indicates whether the toggle switch is selected. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="disabled"
+      defaultValue="false"
+      [isOptional]="true"
+    >
+      Indicates whether to disable the toggle switch. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="tabIndex"
+      defaultValue="0"
+      [isOptional]="true"
+    >
+      Specifies a tab index for the toggle switch. If not defined, the index is set to the position of the toggle switch on load. This property accepts <sky-code>number</sky-code> values.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties
+    sectionHeading="Toggle switch events"
+  >
+    <sky-demo-page-property
+      propertyName="toggleChange"
+      [isOptional]="true"
+    >
+      Fires when the checked state of a toggle switch changes. The event provides an argument of type <sky-code>SkyToggleSwitchChange</sky-code>.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties
+    sectionHeading="SkyToggleSwitchChange type"
+  >
+    <sky-demo-page-property
+      propertyName="checked"
+    >
+      References the <sky-code>checked</sky-code> state of the toggle switch.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-example>
+    <sky-toggle-switch-demo>
+    </sky-toggle-switch-demo>
+    <sky-demo-page-code
+      demoName="Toggle switch"
```

Comment:
The demo seems a little off to me. See my notes below about the first two toggle switches.